components/facebook_groups/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started
+In order to connect your Facebook Group to Pipedream, you'll need to navigate to the group which you are an administrator of, and in the lefthand sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the edit button under **Apps**.
```

Comment:
```suggestion
In order to connect your Facebook Group to Pipedream, you need to navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Reply:
Changed 'you'll need to' to 'you need to' (Google's Developer docs style guide: contractions), Changed 'lefthand' to 'left' (Dreyer's English: simplicity)